runner.go

```diff
@@ -107,6 +109,19 @@ func Run(options ...RunOption) {
 		return
 	}
 
+	if describeTasks {
+		var outputString string
+		for _, task := range tasks {
+			if len(task.Description) > 0 {
+				outputString = fmt.Sprintf("%s\n\t%s\t # %s", outputString, task.Name, task.Description)
```

Comment:
Is the idea here that by adding the comment we are maintaining compatibility with fzf?

Reply:
I was hoping to maintain compatibility with fzf, but hashes quotes and semicolons were not enough to prevent taskrunner from trying to run every word in the comment as a task. I couldn't think of anything else that seemed promising. Would you like me to change the formatting?